autogluon_zeroshot/contexts/context_2022_10_13.py

```diff
@@ -1,74 +1,42 @@
-    """
-    :param folds:
-    :param load_zeroshot_pred_proba:
-    :param lazy_format: whether to load with a format where all data is in memory (`TabularPicklePredictions`) or a
-    format where data is loaded on the fly (`TabularPicklePerTaskPredictions`). Both formats have the same interface.
-    :return:
-    """
-    if folds is None:
-        folds = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
-
-    df_results, df_results_by_dataset, df_raw, df_metadata = load_results(
-        results=str(Paths.all_v3_results_root / "results_ranked_valid.csv"),
-        results_by_dataset=str(Paths.all_v3_results_root / "results_ranked_by_dataset_valid.parquet"),
-        raw=str(Paths.all_v3_results_root / "openml_ag_2022_10_13_zs_models.parquet"),
-        metadata=str(Paths.data_root / "metadata" / "task_metadata.csv"),
-    )
-    df_results_by_dataset = combine_results_with_score_val(df_raw, df_results_by_dataset)
-
-    # Load in real framework results to score against
-    path_prefix_automl = Paths.results_root / 'automl'
-    df_results_by_dataset_automl = load_pd.load(f'{path_prefix_automl}/results_ranked_by_dataset_valid.csv')
-
-    zsc = ZeroshotSimulatorContext(
-        df_results_by_dataset=df_results_by_dataset,
-        df_results_by_dataset_automl=df_results_by_dataset_automl,
-        df_raw=df_raw,
-        folds=folds,
-    )
-
-    configs_prefix_1 = str(Paths.data_root / 'configs/configs_20221004')
-    configs_prefix_2 = str(Paths.data_root / 'configs')
-    config_files_to_load = [
-        f'{configs_prefix_1}/configs_catboost.json',
-        f'{configs_prefix_1}/configs_fastai.json',
-        f'{configs_prefix_1}/configs_lightgbm.json',
-        f'{configs_prefix_1}/configs_nn_torch.json',
-        f'{configs_prefix_1}/configs_xgboost.json',
-        f'{configs_prefix_2}/configs_rf.json',
-        f'{configs_prefix_2}/configs_xt.json',
-        f'{configs_prefix_2}/configs_knn.json',
-    ]
-    configs_full = load_configs(config_files_to_load)
-
-    zeroshot_pred_proba = None
-    zeroshot_gt = None
-    if load_zeroshot_pred_proba:
-        path_zs_gt = str(Paths.all_v3_results_root / 'zeroshot_gt_2022_10_13_zs.pkl')
-        pred_pkl_path = Paths.all_v3_results_root / 'zeroshot_pred_proba_2022_10_13_zs.pkl'
-        zeroshot_gt = zsc.load_groundtruth(path_gt=path_zs_gt)
-        zeroshot_pred_proba = zsc.load_pred(
-            pred_pkl_path=pred_pkl_path,
-            lazy_format=lazy_format,
-        )
-
-        # keep only dataset whose folds are all present
-        intersect_folds_and_datasets(zsc, zeroshot_pred_proba, zeroshot_gt)
-        zeroshot_pred_proba.restrict_models(zsc.get_configs())
-
-    return zsc, configs_full, zeroshot_pred_proba, zeroshot_gt
-
-
+from .context import BenchmarkContext
+from ..loaders import Paths
+
+
+s3_download_map = {
+    'results/all_v3/zeroshot_gt_2022_10_13_zs.pkl': 's3://automl-benchmark-ag/aggregated/ec2/2022_10_13_zs/zeroshot_gt_2022_10_13_zs.pkl',
```

Comment:
`s3://automl-benchmark-ag/aggregated/ec2` appears 23 times in the code, it would make a lot of sense to just import it from a single location.

Reply:
I generated these mappings automatically via some other code, so I'd prefer to keep as is, given that this was not manually created.